api_client/python/timesketch_api_client/analyzer.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2020 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Timesketch API analyzer result object."""
+from __future__ import unicode_literals
+
+import datetime
+import json
+
+from . import definitions
+from . import resource
+
+
+class AnalyzerResult(resource.BaseResource):
+    """Class to store and retrieve session information for an analyzer."""
+
+    def __init__(self, timeline_id, session_id, sketch_id, api):
+        """Initialize the class."""
+        self._session_id = session_id
+        self._sketch_id = sketch_id
+        self._timeline_id = timeline_id
+        resource_uri = (
+            '{0:s}/sketches/{1:d}/timelines/{2:d}/analysis/').format(
+                api.api_root, sketch_id, timeline_id)
+        super(AnalyzerResult, self).__init__(api, resource_uri)
+
+    def _fetch_data(self):
+        """Returns..."""
+        response = self.api.session.get(self.resource_uri)
+        if not response.status_code in definitions.HTTP_STATUS_CODE_20X:
+            return {}
+
+        data = response.json()
+
+        objects = data.get('objects')
+        if not objects:
+            return {}
+
+        for result in objects[0]:
+            result_id = result.get('analysissession_id', -1)
```

Comment:
-1 ?

Reply:
Why not None?

Reply:
No particular reason, other than to get an integer, might as well just have it default to none